scss/detail.scss

```
@@ -1,22 +1,85 @@
+// variables
+$container-height: 416px;
+$container-width: 900px;
+$badge-height: 119px;
+$badge-width: 95px;
+
+.back-arrow {
+  display: flex;
+  align-items: center;
+  width: 90%;
+  height: 20px;
+  margin-bottom: 33px;
+  cursor: pointer;
+  
+  h4::before {
```

Since the h4 has the class `text-back-arrow` assigned to it
```suggestion
  .text-back-arrow::before {
```